srcs/utils/logging.cpp

```
@@ -2,6 +2,13 @@
 #include <iomanip>
 #include <sstream>
 
+LogLevel LOG_LEVEL = INFO;
+
+void setLogLevel(LogLevel level)
```

cppcheck

[style] The function 'setLogLevel' is never used.

(unusedFunction)